src/poliastro/maneuver.py

```diff
@@ -48,6 +50,9 @@ def __init__(self, *args):
         except TypeError:
             raise ValueError("Delta-V must be three dimensions vectors")
 
+    def __repr__(self):
+        return "Number of impulses: %s, Total cost: %s" % (len(self.impulses), self.get_total_cost().round(6))
```

Comment:
Instead of the old formatting, let's use the new one, so we can easily transition to f-strings when we drop Python 3.6, take a look at https://pyformat.info/

In particular, you should remove `.round(6)` and use https://pyformat.info/#number_padding instead.

Reply:
I tried number padding but it fails because `get_total_cost()` returns a Quantity object.
```
  File "/home/uchiha/Desktop/poliastro/src/poliastro/maneuver.py", line 52, in __repr__
    return "Number of impulses: {}, Total cost: {.6f}" .format(len(self.impulses), self.get_total_cost())
  File "/home/uchiha/Desktop/poliastro/venv/lib/python3.7/site-packages/astropy/units/quantity.py", line 835, in __getattr__
    attr))
AttributeError: 'Quantity' object has no '6f' member
```

Reply:
https://docs.astropy.org/en/stable/api/astropy.units.quantity.Quantity.html#astropy.units.quantity.Quantity.round

Reply:
Because the syntax is incorrect, let me show you:

```python
>>> q = 1 * u.km
>>> "{.6f}".format(q)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/juanlu/.pyenv/versions/poliastro38/lib/python3.8/site-packages/astropy/units/quantity.py", line 814, in __getattr__
    raise AttributeError(
AttributeError: 'Quantity' object has no '6f' member
>>> "{:.6f}".format(q)
'1.000000 km'
```

Reply:
Okay! Thanks:)